scipy/stats/_qmc.py

```diff
@@ -940,56 +940,61 @@ def reset(self):
         return self
 
 
-class OptimalDesign(QMCEngine):
-    """Optimal design.
+class OptimalLatinHypercube(QMCEngine):
+    """Optimal Latin hypercube sampling (OLHS).
 
-    Optimize the design by doing random permutations to lower the centered
-    discrepancy [1]_.
-
-    The specified optimization `method` is used to select a new set of
-    permutations to perform. If `method` is None, *basinhopping* optimization
-    is used. `niter` set of permutations are performed.
-
-    Centered discrepancy-based design shows better space filling robustness
-    toward 2D and 3D subprojections. Distance-based design shows better space
-    filling but less robustness to subprojections [2]_.
+    Optimize the sample by doing random permutations of coordinates to lower
+    the centered discrepancy [1]_. The resulting sample is still a LHS.
 
     Parameters
     ----------
     d : int
         Dimension of the parameter space.
-    start_design : array_like (n, d), optional
-        Initial design of experiment to optimize. `OrthogonalLatinHypercube`
+    start_sample : array_like (n, d), optional
+        Initial sample to optimize. `LatinHypercube`
         is used to generate a first design otherwise.
     niter : int, optional
-        Number of iterations to perform. Default is 1.
-    method : callable ``f(func, x0, bounds)``, optional
-        Optimization function used to search new samples. Default to
-        *basinhopping* optimization.
+        Number of iterations of optimization to perform. Default is 1.
     seed : {None, int, `numpy.random.Generator`}, optional
         If `seed` is None the `numpy.random.Generator` singleton is used.
         If `seed` is an int, a new ``Generator`` instance is used,
         seeded with `seed`.
         If `seed` is already a ``Generator`` instance then that instance is
         used.
 
+    Notes
+    -----
+    The specified optimization `method` is used to select a new set of
+    permutations to perform. If `method` is None,
+    `~scipy.optimize.dual_annealing` optimization is used. `niter` set of
+    optimizations are performed.
```

Comment:
I thought you removed `method` above?

Reply:
Yes, yes, as I said I did not touch yet the docstrings too much as I am rewriting things.